client/my-sites/plans-grid/components/comparison-grid/index.tsx

```diff
@@ -362,6 +362,7 @@ type ComparisonGridHeaderCellProps = ComparisonGridHeaderProps & {
 	isLargeCurrency: boolean;
 	isPlanUpgradeCreditEligible: boolean;
 	planSlug: PlanSlug;
+	freeTrialPlanSlug?: PlanSlug;
```

Comment:
I know you are probably following along to how `planSlug` is passed, but passing `freeTrialPlanSlug` around is unnecessary. I think you can remove it entirely and retrieve it from the plans in context e.g. see: https://github.com/Automattic/wp-calypso/blob/trunk/client/my-sites/plans-grid/components/actions.tsx#L388-L392

(we may revise the use of context down the line, but it will likely be a replacement with a data-store, not component props)